orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -342,6 +368,22 @@ func (m *swiftDialogMDMMigrator) renderMigration() error {
 			return nil
 		}
 
+		// If we have the migration file and this is a manual migration, we should just send the
+		// user straight to the My device page
+
+		switch migrationType {
+		case constant.MDMMigrationTypeManual:
+			// The migration file only exists if we successfully hit the webhook
+			log.Info().Msg("showing instructions")
+
+			if err := m.handler.ShowInstructions(); err != nil {
+				return err
+			}
+			return nil
+		case constant.MDMMigrationTypeADE:
+		default:
+		}
+
```

Comment:
found this a bit confusing, any reason not to simply

```go
	if migrationType == constant.MDMMigrationTypeManual {
		log.Info().Msg("showing manual migration instructions")
		return m.handler.ShowInstructions()
	}
```

Reply:
Ah yeah, I think I just had this set up for when I do add in the ADE functionality. I can remove it for now tho!

Reply:
ahh, ok that makes sense